CHANGELOG.md

```diff
@@ -7,13 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.28.0] - 2024-08-21
+## [0.29.0] - 2024-08-28
 
 ### Forge
 
 #### Added
 
-- support for Scarb features in `snforge test` - flags the same as in Scarb. Read more [here](https://docs.swmansion.com/scarb/docs/reference/conditional-compilation.html#features)
+- Support for Scarb features in `snforge test` - flags the same as in Scarb. Read more [here](https://docs.swmansion.com/scarb/docs/reference/conditional-compilation.html#features)
+
+#### Changed
+
+- `snforge init` no longer emits warnings
+
```

Comment:
Nit; we are a bit inconsistent with this, but we do not need an empty line here I think.